README.md

```diff
@@ -21,11 +21,15 @@ Future support is planned for the following operations.
 4. Grid-based indexing for points and polygons
 5. Quadtree-based indexing for large-scale point data
 6. R-Tree-based indexing for Polygons/Polylines
- 
-## Compile / Install C++ backend
-To compile and run cuSpatial, use the following steps.
 
-## Install dependencies
+## Install from Conda
+To insall via conda, please run:
+`conda install -c rapidsai-nightly cuspatial`
+
+## Install from Source
+To compile and install cuSpatial's C++ backend, please follow the following steps:
+
+### Install dependencies
 
 Currently, building cuSpatial requires a source installation of cuDF. Install
 cuDF by following the [instructions](https://github.com/rapidsai/cudf/blob/branch-0.10/CONTRIBUTING.md#script-to-build-cudf-from-source)
```

Comment:
This is out of date too, I think. @kkraus14 what is the procedure for building with the cuDF conda package now?

Reply:
I'll wait for the answer form @kkraus14 before resolving

Reply:
as @beckernick state here, https://github.com/rapidsai/cuspatial/pull/53#issuecomment-532369072, someone acutely familiar with the build can fix the README later.   I'll update the conda install portion to include other packages form rapidsai-nightly.  There are currently no conda install options in rapidsai